src/main/resources/db/migration/V4.13.0.1__3575-usernames.sql

```diff
@@ -1 +1 @@
-CREATE UNIQUE INDEX index_authenticateduser_lower_useridentifier ON authenticateduser (lower(useridentifier));
+CREATE UNIQUE INDEX IF NOT EXISTS index_authenticateduser_lower_useridentifier ON authenticateduser (lower(useridentifier));
```

Comment:
FYI - changing these files after someone has deployed them breaks things as Flyway checks the hash of the file and sees that it is different.